Update nodejs (5.2.x), and npm (3.3.x)

We need to [update](https://github.com/puppet-community/puppet-nodejs#beginning-with-nodejs) our nodejs, and npm (on both `Puppetfile` instances), if we would like to eliminate the following error traceback:

```bash
vagrant@vagrant-ubuntu-trusty-64:/vagrant/src/jsx$ npm install babel-core --save
npm WARN deprecated This version of npm lacks support for important features,
npm WARN deprecated such as scoped packages, offered by the primary npm
npm WARN deprecated registry. Consider upgrading to at least npm@2, if not the
npm WARN deprecated latest stable version. To upgrade to npm@2, run:
npm WARN deprecated
npm WARN deprecated   npm -g install npm@latest-2
npm WARN deprecated
npm WARN deprecated To upgrade to the latest stable version, run:
npm WARN deprecated
npm WARN deprecated   npm -g install npm@latest
npm WARN deprecated
npm WARN deprecated (Depending on how Node.js was installed on your system, you
npm WARN deprecated may need to prefix the preceding commands with `sudo`, or if

npm WARN deprecated on Windows, run them from an Administrator prompt.)
npm WARN deprecated
npm WARN deprecated If you're running the version of npm bundled with
npm WARN deprecated Node.js 0.10 LTS, be aware that the next version of 0.10 LTS

npm WARN deprecated will be bundled with a version of npm@2, which has some small
npm WARN deprecated backwards-incompatible changes made to `npm run-script` and
npm WARN deprecated semver behavior.
npm WARN package.json reactjs@1.0.0 No repository field.
npm ERR! Error: UNKNOWN, symlink '../babylon/bin/babylon.js'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Linux 3.13.0-55-generic
npm ERR! command "/usr/bin/node" "/usr/bin/npm" "install" "babel-core" "--save"
npm ERR! cwd /vagrant/src/jsx
npm ERR! node -v v0.10.41
npm ERR! npm -v 1.4.29
npm ERR! path ../babylon/bin/babylon.js
npm ERR! code UNKNOWN
npm ERR! errno -1
npm ERR! not ok code 0
vagrant@vagrant-ubuntu-trusty-64:/vagrant/src/jsx$
```

**Note:** the above traceback result of `npm install babel-core --save` is motivated with the intention of storing the necessary reference in `package.json`, so it can be built on a future iteration.

We should be able to update nodejs to 5.x, which will allow us to attain npm 3 (available on nodejs 0.8).

--- a/puppet/Puppetfile
+++ b/puppet/Puppetfile
@@ -17,7 +17,7 @@ mod 'apt',
 ## Install Module: nodejs
 mod 'nodejs',
   :git => 'git@github.com:puppet-community/puppet-nodejs.git',
-  :ref => '1.1.0'
+  :ref => '1.2.0'
 
 ## Install Module: git
 mod 'git',

--- a/puppet/manifests/install_packages.pp
+++ b/puppet/manifests/install_packages.pp
@@ -1,7 +1,10 @@
 ## include puppet modules: this (also) runs 'apt-get update'
 include python
 include apt
-include nodejs
+
+class { 'nodejs':
+  repo_url_suffix => 'node_5.x',
+}
 
 ## variables
 case $::osfamily {

